src/main/java/org/opentripplanner/updater/traficstreetupdater/traficupdater.java

```diff
@@ -0,0 +1,60 @@
+package org.opentripplanner.updater.traficstreetupdater;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.opentripplanner.graph_builder.linking.TemporaryStreetSplitter;
+import org.opentripplanner.graph_builder.module.time.EdgeLine;
+import org.opentripplanner.graph_builder.module.time.TimeTable;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.updater.GraphUpdaterManager;
+import org.opentripplanner.updater.PollingGraphUpdater;
+
+
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.concurrent.Executor;
+
+
+public class traficupdater  extends PollingGraphUpdater {
+    private  Graph graph;
+    GraphUpdaterManager graphUpdaterManager;
```

Comment:
`private`

Reply:
jak zminie to na private to sie nie kompiluje pozatym u was updater tez jest public